data/858/635/11/85863511.geojson

```diff
@@ -6,30 +6,32 @@
     "edtf:inception":"uuuu",
     "geom:area":0.0,
     "geom:area_square_m":0.0,
-    "geom:bbox":"-4.380798,36.742948,-4.380798,36.742948",
-    "geom:latitude":36.742948,
-    "geom:longitude":-4.380798,
+    "geom:bbox":"-4.373589,36.734557,-4.373589,36.734557",
+    "geom:latitude":36.734557,
+    "geom:longitude":-4.373589,
     "gn:id":"",
     "iso:country":"ES",
-    "lbl:latitude":36.742948,
-    "lbl:longitude":-4.380798,
+    "lbl:latitude":36.734557,
+    "lbl:longitude":-4.373589,
     "mps:latitude":36.749681,
     "mps:longitude":-4.38419,
     "mz:hierarchy_label":0,
     "mz:is_current":-1,
     "mz:is_funky":0,
     "mz:is_hard_boundary":0,
     "mz:is_landuse_aoi":0,
-    "mz:is_official":0,
+    "mz:is_official":1,
     "mz:max_zoom":18.0,
-    "mz:min_zoom":16.0,
+    "mz:min_zoom":14.0,
+    "mz:note":"story of name: http://www.laopiniondemalaga.es/malaga/2008/11/19/historia-ventorrillo-mosca/218641.html",
     "mz:tier_locality":3,
     "mz:tier_metro":30,
     "name:spa_x_preferred":[
         "La Mosca"
     ],
     "name:spa_x_variant":[
-        "La mosca y la sopa"
+        "La mosca y la sopa",
+        "Ventorrillode la Mosca"
```

Comment:
Please verify we're doing the preferred and variant names correctly here, paste in screenshot of Nat's data record in QGIS.